test/Field.test.tsx

```diff
@@ -213,6 +213,10 @@ describe('A <Field />', () => {
                 const { handleBlur, handleChange } = formikProps;
                 expect(field.name).toBe('name');
                 expect(field.value).toBe('jared');
+                expect(field.hasOwnProperty('error')).toBe(true);
+                expect(field.hasOwnProperty('touched')).toBe(true);
+                expect(field.error).toBe(undefined);
+                expect(field.touched).toBe(undefined);
```

Comment:
Is there a better way to test this? I didn't see any test data with values for `error` and `touched`